packages/mobile/src/components/challenge-rewards-drawer/ChallengeRewardsDrawerContent.tsx

```diff
@@ -117,20 +107,6 @@ export const ChallengeRewardsDrawerContent = ({
     claimStatus === ClaimStatus.WAITING_FOR_RETRY
   const claimError = claimStatus === ClaimStatus.ERROR
 
-  const hasCompleted =
```

Comment:
i think airdrop depends on this?

Reply:
actually i removed the "box-in-box" UI of the challenge drawer, but i realized then i also need to add the new "complete" state for the airdrop modal..

Reply:
will do that in follow-up PR